### DISCUSSION TITLE
Can't get Remux running / how does rm2fb interact with this when using a RM2?

### DISCUSSION THREAD

**Discussion by koshibb** [toltec-dev/toltec#547] [Support] [answered]
So I am attempting to install remux now, and I can't seem to get it running. I have installed the opkg, I see it installed status when doing an info call + I have it started using the ssh window. But I am attempting to swipe up to the side and it's not working. 

I notice on remux's website it says it interacts with the rm2fb opkg, but I can't find that one. Another user in a separate discussion that rm2fb is now included by default as part of the 'display' component of toltec. Is there something else I am doing wrong?

**Comment by raisjn** [comment]
how did you start it? systemctl?

and what is your is os version?

**Comment by koshibb** [comment]
Yes `systemctl enable --now remux` im on Version 2.11.0.442

**Reply by Eeems** [reply]
What does `systemctl status remux` output?

**Reply by raisjn** [reply]
can also look at the readme for remux to see a gif of the swipe gestures

if you run "echo show > /run/remux.api", does it show up?

**Comment by koshibb** [comment, marked as answer]
I think I figured it out. It was running this whole time as the ssh said, but activating it wasn't as clear as it should have been. 
On RM2 you have to swipe up FROM the edge of the screen. i.e. finger on bottom right, up to top right, to activate it.  On the wiki it was talking about swiping up to the side and I just took that the wrong way!
